extensions/arc/src/localizedConstants.ts

```diff
@@ -122,7 +122,8 @@ export const confirmNewPassword = localize('arc.confirmNewPassword', "Confirm th
 export const learnAboutPostgresClients = localize('arc.learnAboutPostgresClients', "Learn more about Azure PostgreSQL Hyperscale client interfaces");
 export const scalingCompute = localize('arc.scalingCompute', "scaling compute vCores and memory.");
 export const computeAndStorageDescriptionPartOne = localize('arc.computeAndStorageDescriptionPartOne', "You can scale your Azure Arc enabled");
-export const computeAndStorageDescriptionPartTwo = localize('arc.computeAndStorageDescriptionPartTwo', "PostgreSQL Hyperscale server group by");
+export const postgresComputeAndStorageDescriptionPartTwo = localize('arc.postgres.computeAndStorageDescriptionPartTwo', "PostgreSQL Hyperscale server group by");
+export const miaaComputeAndStorageDescriptionPartTwo = localize('arc.computeAndStorageDescriptionPartTwo', "SQL managed instance - Azure Arc by");
```

Comment:
Are you missing 'group' before 'by', if not this string does not sound complete.

Reply:
Huh? Look at the screenshot - it's a sentence that's built up from multiple parts. This isn't ideal by far but we don't currently have the ability to embed links into a text component so I have to split it up like this. 

`You can scale your Azure SQL managed instance - Azure Arc by scaling compute vCores and memory.`